modules/google_sql_database_instance/variables.tf

```diff
@@ -132,10 +132,14 @@ variable "settings_backup_configuration_backup_retention_settings_retained_backu
   default     = 7
 }
 
-variable "settings_ip_configuration_require_ssl" {
-  description = "(Optional) Whether SSL connections over IP are enforced or not."
-  type        = bool
-  default     = true
+variable "settings_ip_configuration_ssl_mode" {
+  description = "(Optional) Specify how SSL connection should be enforced in DB connections."
```

Comment:
Nice to list the supported values in the description so your editor gives them to you when using the module.